src/igvfd/schemas/changelogs/matrix_file.md

```diff
@@ -2,6 +2,8 @@
 
 ### Minor changes since schema version 4
 
+* Expand `file_format` enum list to include `hic`.
```

Comment:
Please change to "Add `hic` to `file_format`.
Add  `genomic position` to `dinemsion1`, `dinemsion2`.

Reply:
This wording is consistent with how we've referred to adding to enums before.

ex.
"### Minor changes since schema version 1
* Expand `collections` enum list to include `ClinGen`, `GREGoR`, `IGVF_catalog_beta_v0.1`, and `MaveDB`."

Reply:
I think both wordings are fine, we should just be consistent. Maybe there should be some ruleset for changelog wording because typically I just rely on how prior entries worded similar changes.